tensorflow/lite/delegates/webnn/webnn_delegate.cc

```diff
@@ -62,6 +62,7 @@ class Delegate {
         {0, "Default"}, {1, "GPU"}, {2, "CPU"}};
     std::unordered_map<uint32_t, std::string> power_preference_names = {
         {0, "Default"}, {1, "High-performance"}, {2, "Low-power"}};
+    instance_ = std::make_unique<webnn_native::Instance>();
```

Comment:
@huningxin, this would throw `use of undeclared identifier 'webnn_native'` since we don't include `<webnn_native/WebnnNative.h>` for Emscripten build.